jbpm-case-mgmt/jbpm-case-mgmt-cmmn/src/main/resources/META-INF/DI.xsd

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" 
+            xmlns:dc="http://www.omg.org/spec/CMMN/20151109/DC" 
+            xmlns:di="http://www.omg.org/spec/CMMN/20151109/DI"
+            targetNamespace="http://www.omg.org/spec/CMMN/20151109/DI"
+            elementFormDefault="qualified" 
+            attributeFormDefault="unqualified">
+	<xsd:import namespace="http://www.omg.org/spec/CMMN/20151109/DC" 
+                    schemaLocation="DC.xsd"/>

```

Comment:
One more thing I forgot. I see these schemas in both src/main/resources/META-INF and src/main/xsd. Why is it needed to have it duplicated? Because if I found it correctly, ExtensibleXMLParser in Drools looks for it in both directories anyway.

Reply:
The xsd location is explicitly for OSGi